project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -0,0 +1,201 @@
+  // Message will be saved in message_notify_send_message().
+  message_notify_send_message($wrapper->value());
+
+}
+
+/**
+ * Email notification to group admins about new member at moderated group.
+ */
+function c4m_user_notifications_moderated_group_notify_admins_membership_accepted($membership) {
+}
+
+/**
+ * Email notification to user about acceptance to open group.
+ */
+function c4m_user_notifications_open_group_notify_user_membership_accepted($membership) {
+}
+
+/**
+ * Email notification to group admins about new member at open group.
+ */
+function c4m_user_notifications_open_group_notify_admins_membership_accepted($membership) {
+}
+
+/**
+ * Email notification to user about rejection of membership request.
+ */
+function c4m_user_notifications_moderated_group_notify_user_membership_rejected($membership) {
+}
+
+/**
+ * Email notification to group admins about membership cancelation.
+ */
+function c4m_user_notifications_notify_admins_membership_canceled($entity) {
+}
+
+/**
+ * Email notification to user about his membership cancelation.
+ */
+function c4m_user_notifications_notify_user_membership_canceled($entity) {
+}
+
+/**
+ * Email notification about organic group status change to 'published'.
+ */
+function c4m_user_notifications_group_status_set_published($node) {
+}
+
+/**
+ * Email notification about organic group status change to 'republished'.
+ */
+function c4m_user_notifications_group_status_set_republished($node) {
+}
+
+/**
+ * Email notification about organic group status change to 'unpublished'.
+ */
+function c4m_user_notifications_group_status_set_unpublished($node) {
+}
+
+/**
+ * Email notification about organic group status change to 'archived'.
+ */
+function c4m_user_notifications_group_status_set_archived($node) {
+}
+
+/**
+ * Email notification about organic group status change to 'softdelete'.
+ */
+function c4m_user_notifications_group_status_set_softdeleted($node) {
+}
+
+/**
+ * Email notification about organic group deletion.
+ */
+function c4m_user_notifications_group_hard_deleted($node) {
+}
+
+/**
+ * Email notification about organic group access change.
+ */
+function c4m_user_notifications_group_access_changed($node) {
+}
+
+/**
+ * Email notification to previous owner about group organic owner change.
+ */
+function c4m_user_notifications_notify_prev_owner_owner_update($node) {
+}
+
+/**
+ * Email notification to new owner about organic group owner change.
+ */
+function c4m_user_notifications_notify_new_owner_owner_update($node) {
+}
+
+/**
+ * Email notification to group admins about organic group owner change.
+ */
+function c4m_user_notifications_notify_admins_owner_update($node) {
+}
+
+/**
+ * Email notification about user adding a comment.
+ */
+function c4m_user_notifications_notify_user_commented($comment) {
+}
+
+/**
+ * Email notification about user invitation to group, sent by admin.
+ */
+function c4m_user_notifications_notify_user_invited_by_admin(&$form, &$form_state) {
```

Comment:
Can remove, as we do not intend to change the arguments passed.
On 16 Jun 2016 08:54, "ordavidil" <notifications@github.com> wrote:

> In
> project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc
> <https://github.com/capacity4dev/capacity4more/pull/914#discussion_r67290577>
> :
>
> > +/**
> > + * Email notification to group admins about organic group owner change.
> > + */
> > +function c4m_user_notifications_notify_admins_owner_update($node) {
> > +}
> > +
> > +/**
> > + * Email notification about user adding a comment.
> > + */
> > +function c4m_user_notifications_notify_user_commented($comment) {
> > +}
> > +
> > +/**
> > + * Email notification about user invitation to group, sent by admin.
> > + */
> > +function c4m_user_notifications_notify_user_invited_by_admin(&$form, &$form_state) {
>
> @anvmn <https://github.com/anvmn>
> Can we remove the & from the arguments ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/capacity4dev/capacity4more/pull/914/files/4f098598467083e251e46325ae97e95cd9e2fe1d#r67290577>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARDchR4p2K3dU2TgliidZ7MHUljET1oOks5qMOUGgaJpZM4IrrjP>
> .
>